internal/credentials/spiffe_test.go

```diff
@@ -231,3 +233,110 @@ func (s) TestSPIFFEIDFromCert(t *testing.T) {
 		})
 	}
 }
+
+var SPIFFETrustBundle map[string]*spiffebundle.Bundle
+
+func TestKnownSPIFFEBundle(t *testing.T) {
+	bundles, err := LoadSPIFFEBundleMap(testdata.Path("spiffe/spiffebundle.json"))
+	if err != nil {
+		t.Fatalf("Error during parsing: %v", err)
+	}
+	if len(bundles) != 2 {
+		t.Fatal("did not parse correct bundle length")
```

Comment:
Also, please include the actual length as seen by the test. See: go/go-style/decisions#got-before-want